app/pages/api/config.ts

```diff
@@ -1,34 +1,17 @@
-import { NextApiRequest, NextApiResponse } from "next";
-
 import { createConfig } from "@/db/config";
```

Comment:
Nit: `createConfig as createConfigInDatabase`? When reading the code below initially I asked myself "why the config in being created, when we need to sent it to a db?" - maybe it could be worth to mention the db in this context?

Reply:
I thought that the import is coming from a path with "db" in it was suffcient ?

Reply:
I meant that when looking at the logic, not the imports, it wasn't immediately apparent that it creates a config in db, and not just on a client side, but this is a minor thing and the "/db" path should be enough 👍

Reply:
OK, next time maybe I'll write `import * as db from '@/db/config'` so that it's clearer from the call site :)